local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationsPlugin.java

```diff
@@ -1,5 +1,7 @@
 package com.capacitorjs.plugins.localnotifications;
 
+import android.Manifest;
+import android.annotation.TargetApi;
```

Comment:
```suggestion
```

As commented, the TargetApi is not needed, so can be removed and this import too